text/0059-portable-account.md

```diff
@@ -0,0 +1,81 @@
+- FIP: 59
+- title: Portable Account
+- author: Viktor Levente Tóth, @nugaon
+- status: draft
+- created: 2022-10-04
+
+# Summary
+Addressing web3 login workflow where users can retreive their wallet in a decentralized manner from anywhere by providing a _username_ and a _password_.
+
```

Comment:
```suggestion
Addressing web3 login workflow where users can retrieve their wallet in a decentralised manner from anywhere by providing a _username_ and a _password_.
```

Reply:
"decentralized" is a correct spelling as well.